### DISCUSSION TITLE
Vega-lite for Data Studio

### DISCUSSION THREAD

**Discussion by VerlandeRyan** [vega/vega-lite#7742] [Q&A]
Hello !

I'm using a community visualization that allows me to use Vega/Vega-lite on Google Data Studio
However, I think it has been made compatible with v3 only (because when I use a feature available on v4/v5, it doesn't work)
I'd like to know if the developper works on Vega/Vega-Lite here and if it was possible to update the custome component to v5 on Data Studio ?
Or maybe he doesn't work on it, so do you have an idea on how can I contact him (because I don't find any contact email or something) to ask him

I'm sorry if it's not the right place to ask you this question..

Thanks by advance for your help !

P.S : Here's a link to data studio if you want to see what I mean : https://datastudio.google.com/reporting/88379d94-19e1-4403-ae92-d279dcb2cbf5/page/Vuq9

**Comment by domoritz** [comment]
I'm not involved in it. Maybe some knows who is. You could try sending a support request although I know how often you don't get a response.

**Reply by VerlandeRyan** [reply]
Ok thank you
How can I try sending a support request ?

**Reply by kanitw** [reply]
from your link, the term of service page says 

>These terms govern your use of Data Studio Community Visualizations written and deployed by Jerry Chen. By accessing or using these visualizations, you agree to become bound by these terms.

So perhaps, you should find a way to contact the author (Jerry Chen)? 

(Since we do not work on this ourselves, I don't think we can provide a better clue than this.)

**Reply by VerlandeRyan** [reply]
I didn't find any way to contact the author unfortunately... 😕

But thank you anyway 🙂
